lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -681,6 +682,15 @@ sub create_clones {
                     dependency    => OpenQA::JobDependencies::Constants::CHAINED,
                 });
         }
+        for my $p (@{$info->{directly_chained_parents}}) {
+            # normally we don't clone chained parents, but you never know
```

Comment:
A little too defensive against future yous IMO

Reply:
I just wanted the behavior consistent with regularly chained parents because I don't think it should behave differently when it comes to cloning. Or do you really just mean the comment? Then I better change it to `be consistent with regularly chained parents regarding cloning`.

Reply:
I changed the comment text. Unresolve if is is not sufficient.